Sources/PointFree/Bootstrap.swift

```diff
@@ -67,7 +67,7 @@ private let loadEnvVars = { (_: Prelude.Unit) -> EitherIO<Error, Prelude.Unit> i
   )
   Current.mailgun = .init(
     apiKey: Current.envVars.mailgun.apiKey,
-    appSecret: Current.envVars.appSecret,
+    appSecret: Current.envVars.appSecret.rawValue,
```

Comment:
`Mailgun` doesn't know about `EnvVars`, so escape-hatching the initializer here.

Reply:
could we move the tagged `AppSecret` to models so that everyone gets access to it?